src/app/App.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import './App.css';
+import { fetchBookData } from '../api-calls'
+import Sidebar from '../Sidebar/Sidebar'
+import {IState, IList, IBook} from '../Interfaces'
+
+// type State = {
+//   bookLists: List[]
+// }
+
+// type List = {
+//   list_id: number
+//   list_name: string
+//   books: Book[]
+// }
+
+// type Book = {
+//   author: string
+//   book_image: string
+//   description: string
+//   primary_isbn13: string
+//   publisher: string
+//   rank: number
+//   title: string
+// }
+
+class App extends React.Component<{}, IState> {
+    state: IState = {
+      bookLists: []
+    }
+
+  componentDidMount = () => {
+    fetchBookData()
+    .then((data) => { 
+        // console.log(data.results.lists)
+        this.setState({ bookLists: data.results.lists })
+        console.log(this.state.bookLists)
+      })
```

Comment:
Reminder to include a `.catch()`.  This will catch any 500 errors as well as the errors that get thrown if the `response.ok` isn't good!